plio/views.py

```diff
@@ -456,16 +456,22 @@ def metrics(self, request, uuid):
             )
 
             # retain only the responses to items which are questions
-            dff = df[df["item_type"] == "question"].reset_index(drop=True)
-            question_df = dff[dff.survey.eq(False)].reset_index(drop=True)
+            question_df = df[df["item_type"] == "question"].reset_index(drop=True)
+            question_df = question_df[question_df.survey.eq(False)].reset_index(
```

Comment:
it is more common syntax to simply do this and likely more computationally efficient too!
```suggestion
            question_df = question_df[~question_df.survey].reset_index(
```